modules/graph/tools/graph_loader.h

```diff
@@ -60,6 +60,7 @@ struct loader_options {
   bool compact_edges = false;
   bool print_memory_usage = false;
   bool print_normalized_schema = false;
+  bool use_perfect_hash = false;
```

Comment:
Add the docs for `use_prefect_hash` to `modules/graph/README.rst`.

Reply:
Done.